sharktank/sharktank/layers/rotary_embedding.py

```diff
@@ -238,3 +235,17 @@ def _replicate(self, t):
             t = ops.replicate(t, self.tensor_parallelism_size)
 
         return t
+
+
+def compute_rotary_embedding_table(
+    positions: torch.Tensor,
```

Comment:
Nit: would it make more sense to just rename _compute_rotary_embedding_table?

Reply:
I want to use this function outside of  the class.

Reply:
Yes, i mean, instead of:

- copying compute_rotary_embedding_table so we can use it outside the class
- make old _compute_rotary_embedding_table redirect to compute_rotary_embedding_table

Just

- rename _compute_rotary_embedding_table to compute_rotary_embedding_table and use it outside the class
- change all referenes to _compute_rotary_embedding_table to use compute_rotary_embedding_table instead

The latter requires an IDE and is slightly more work, but does not leave a stub/redirect function behind.

Reply:
The class variant got updated to handle the hugging face case so it has more hair now.